package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fh-js-sdk",
-  "version": "3.0.1",
+  "version": "3.0.2",
```

Comment:
Also, please, let me know if I am wrong here, but it is a new feature and not a fixed bug. Am I right? If yes, the bumped version should be 3.1.0 instead of 3.0.1. For further information see the link https://semver.org/.

Reply:
Yeah, this is just a bug fix.